decidim-budgets/db/migrate/20170612101846_migrate_projects_category.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class MigrateProjectsCategory < ActiveRecord::Migration[5.1]
+  def change
+    Decidim::Budgets::Project.find_each do |project|
+      Decidim::Categorization.create!(
+        decidim_category_id: project.category.id,
```

Comment:
Yeah, I think @josepjaume is right. But I think you don't need the (no longer existing) association though. Can't you do `project.decidim_category_id`? Anyways, if you can use plain SQL, I agree it's better.

Rergarding the existence of the column, it still exists right? I mean, you don't remove it until the next statement in the migration... If you mean checking for `nil`, yes, I think you need to skip nil values, whether you chose the activerecord or plain SQL approach.